src/proto/h1/conn.rs

```diff
@@ -601,13 +602,18 @@ where
                             Writing::KeepAlive
                         }
                     }
-                    Err(_not_eof) => Writing::Closed,
+                    Err(_not_eof) => {
+                        return Err(crate::Error::new_user_body(
```

Comment:
Why not just `new_body_write_aborted()`?

Reply:
to get the same `Error` as when user calls `tx.abort()`. with `Err(crate::Error::new_body_write_aborted())`  `Display` is: `body write aborted`, when combining both `Display` is `error from user's HttpBody stream: body write aborted`.
Want me to leave only `new_body_write_aborted()`?

Reply:
Oh, now I see it, my bad. This is right!